receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper_linux.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/shirou/gopsutil/v3/cpu"
 	"go.opentelemetry.io/collector/model/pdata"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/internal"
```

Comment:
Please take a look at the failing tests

Reply:
sorry I run it on a different arch and the tests weren't passing automatically after the push. I change it